### ISSUE TITLE
Room booking: booking for multi-day events not properly handled

### ISSUE THREAD

**Issue by jouvin** [indico/indico#4088]
If you create a multi-day event in Indico, select a room for it (managed by the room booking) but don't book it at event creation, you get a button to do it in the event settings. If the event has no timetable, you have a book button that allows to book the selected room for the whole event. If you define a timetable, it is no longer the case and I found 2 problems:

- If you book for the whole event, without selecting a sesssion and/or contribution, the time slot is defined to the slot already defined in the timetable, not the slot defined in the event settings.  It is unexpected as you may do the booking before having your full timetable.

- If you book for the whole event, you are asked to select one of the days and there is no 'All days' entry. It is very unpractical and lead the the creation of different "disconnected" bookings rather than one booking matching the event duration. It is potentially misleading.

**Comment by ThiefMaster** [comment]
I don't think there's any perfect way of handling this for multi-day events... we can't know how long each day lasts (unless there's a full timetable for that day)...

**Comment by jouvin** [comment]
I think that the if the booking is done from the event, the booking time slot should be the time slot defined in the event settings (which is mandatory if I'm right). I agree that there is no perfect guess but it seems to me to be the less wrong... and it would be consistent with what is done for single day events.

**Comment by ThiefMaster** [comment]
That doesn't work if you have e.g. an event starting at 6pm and ending at 2pm. Unlike room bookings, events are not every day from the start time to the end time, but from the start time on the first day to the end time on the last day.

**Comment by jouvin** [comment]
Agreed. So from your example, I'd book the room from 6 pm the first day, the full day the other days and up to the ending time the last day... It would be a better default. Or you should explicitely raise a warning that selecting a room for a multi-day event should not be done before the timetable is complete. Else it is very confusing...
